core/modules/replica/ClusterHealthJob.cc

```diff
@@ -147,13 +147,43 @@ ClusterHealth const& ClusterHealthJob::clusterHealth() const {
             context() + "clusterHealth  can't use this operation before finishing the job");
 }
 
+
 std::list<std::pair<std::string,std::string>> ClusterHealthJob::extendedPersistentState() const {
     std::list<std::pair<std::string,std::string>> result;
     result.emplace_back("timeout_sec", std::to_string(timeoutSec()));
     result.emplace_back("all_workers", allWorkers() ? "1" : "0");
     return result;
 }
 
+
+std::list<std::pair<std::string,std::string>> ClusterHealthJob::persistentLogData() const {
+
+    std::list<std::pair<std::string,std::string>> result;
+
+    auto&& health = clusterHealth();
+
+    for (auto&& entry: health.qserv()) {
+
```

Comment:
is this an extra (incorrect) line inside the for block? You do it again below, so I think it's intentional. I would not put it there (IMO), but does our coding standard say anything about it?

Reply:
I don't think we have any specific coding standard for empty lines within the bodies of methods.